.github/workflows/release.yaml

```diff
@@ -1,15 +1,17 @@
 name: "Release"
 
 on:
-  push:
-    branches:
-      - master
+  workflow_run:
+    workflows: ["Test"]
+    types: [completed]
+    branches: [master]
```

Comment:
Nit, do we need arrays here if it's just a single value?

Reply:
@danielbate I thought we did, but we don't. Changed